docs/getting_started/install_using_ddev.md

```diff
@@ -0,0 +1,384 @@
+sed -i 's/ibexa_params.d/sites-enabled\/ibexa_params.d/' .ddev/nginx_full/ibexa.conf;
+```
+
+<!--
+!!! note
+
+    To replace `%BINARY_DATA_HANDLER%` with `dfs` is only needed if you plan to [share binary files](../infrastructure_and_maintenance/clustering/clustering_using_ddev.md) like a multi-frontend cluster does.
+-->
+
+#### Apache Virtual Host
+
+To set the Apache Virtual Host, `.ddev/apache/apache-site.conf` will be override with Ibexa DXP's config. It can be done manually or using a script.
+
+##### Manual
+
+Copy the Virtual Host template as the new Apache configuration:
+
+```bash
+cp vendor/ibexa/post-install/resources/templates/apache2/vhost.template .ddev/apache/apache-site.conf
+```
+
+Then, replace the placeholders with the appropriate values in this `.ddev/apache/apache-site.conf`:
+
+| Placeholder         | Value                        |
+|---------------------|------------------------------|
+| `%IP_ADDRESS%`      | `*`                          |
+| `%PORT%`            | `80`                         |
+| `%HOST_NAME%`       | `my-ddev-project.ddev.site`  |
+| `%HOST_ALIAS%`      | `*.ddev.site`                |
+| `%BASEDIR%`         | `/var/www/html`              |
+| `%BODY_SIZE_LIMIT%` | `0`                          |
+| `%TIMEOUT%`         | `0`                          |
+| `%FASTCGI_PASS%`    | `unix:/var/run/php-fpm.sock` |
+
+For example, it can be done with `sed`:
+
+```bash
+sed -i 's/%IP_ADDRESS%/*/' .ddev/apache/apache-site.conf
+sed -i 's/%PORT%/80/' .ddev/apache/apache-site.conf
+sed -i 's/%BASEDIR%/\/var\/www\/html/' .ddev/apache/apache-site.conf
+sed -i 's/%BODY_SIZE_LIMIT%/0/' .ddev/apache/apache-site.conf
+sed -i 's/%TIMEOUT%/0/' .ddev/apache/apache-site.conf
+sed -i 's/%FASTCGI_PASS%/unix:\/var\/run\/php-fpm.sock/' .ddev/apache/apache-site.conf
+```
+
+Finally, restart the project:
+
+```bash
+ddev restart
+```
+
+##### Scripted
+
+Generate the Virtual Host using https://github.com/ibexa/docker/blob/main/scripts/vhost.sh:
+
+```bash
+curl -O https://raw.githubusercontent.com/ibexa/docker/main/scripts/vhost.sh
+bash vhost.sh --template-file=vendor/ibexa/post-install/resources/templates/apache2/vhost.template \
+  --ip='*' \
+  --host-name='my-ddev-project.ddev.site' \
+  --host-alias='*.ddev.site' \
+  --basedir='/var/www/html' \
+  --sf-env=dev \
+  > .ddev/apache/apache-site.conf
+rm vhost.sh
+```
+
+Then, restart the project:
+
+```bash
+ddev restart
+```
+
+### Run an already existing project
+
+- Instead of creating an empty directory like in _[1. Create a DDEV project directory](#1-create-a-ddev-project-directory)_, a directory already containing an Ibexa DXP project is used.
+- In _[2. Configure DDEV / Configure PHP version and document root](#configure-php-version-and-document-root)_, no need to create the Document root, `--create-docroot` mustn't be used.
+- Instead of `ddev composer create` from _[5. Create Ibexa DXP project](#5-create-ibexa-dxp-project)_, use solely `ddev composer install`.
```

Comment:
```suggestion
- Instead of `ddev composer create-project` from _[5. Create Ibexa DXP project](#5-create-ibexa-dxp-project)_, use solely `ddev composer install`.
```

Reply:
idem, `ddev composer create-project` is not supported. Maybe `ddev exec composer create-project` but I never tested it preferring the buit-in short-cut.